src/requirements/data/majors/hd.ts

```diff
@@ -0,0 +1,263 @@
+    ],
+    fulfilledBy: 'courses',
+    perSlotMinCount: [2],
+  },
+  {
+    name: 'Humanities',
+    description: 'Any course with the course distribution HA, LA, or CA.',
+    source:
+      'https://www.human.cornell.edu/sites/default/files/Academics/Registrar/Curriculum%20sheets/2020-2021/HD%20Curriculum%20Sheet%202020-2021.pdf',
+    checker: [
+      (course: Course): boolean =>
+        (course.catalogDistr?.includes('CA-') ?? false) ||
+        (course.catalogDistr?.includes('HA-') ?? false) ||
+        (course.catalogDistr?.includes('LA-') ?? false),
+    ],
+    fulfilledBy: 'courses',
+    perSlotMinCount: [1],
+  },
+  {
+    name: 'Statistics',
+    description:
+      'A statistics course taken at Cornell (AP Statistics is not accepted). PSYCH 2500 is strongly recommended for HD majors, and must be taken by students planning to complete the Honors Program.',
+    source:
+      'https://www.human.cornell.edu/sites/default/files/Academics/Registrar/Curriculum%20sheets/2020-2021/HD%20Curriculum%20Sheet%202020-2021.pdf',
+    checker: includesWithSingleRequirement(
+      'AEM 2100',
+      'BTRY 3010',
+      'ILRST 2100',
+      'STSCI 2100',
+      'MATH 1710',
+      'PAM 2100',
+      'PSYCH 2500'
+    ),
+    fulfilledBy: 'courses',
+    perSlotMinCount: [1],
+  },
+  {
+    name: 'Natural Science I',
+    description:
+      'One of the following natural science courses taken at Cornell, or a score of 5 on AP Biology. No lab is required.',
+    source:
+      'https://www.human.cornell.edu/sites/default/files/Academics/Registrar/Curriculum%20sheets/2020-2021/HD%20Curriculum%20Sheet%202020-2021.pdf',
+    checker: includesWithSingleRequirement('BIOG 1140', 'BIOMG 1350', 'BIOG 1440', 'BIOG 1445'),
+    fulfilledBy: 'courses',
+    perSlotMinCount: [1],
+  },
+  {
+    name: 'Natural Science II',
+    description:
+      'Any 3-4 credit course with a Course Distribution PBS, BIOLS-AG, or BIONLS-AG. Course must be taken at Cornell (no AP credit allowed).',
+    source:
+      'https://www.human.cornell.edu/sites/default/files/Academics/Registrar/Curriculum%20sheets/2020-2021/HD%20Curriculum%20Sheet%202020-2021.pdf',
+    checker: [
+      (course: Course): boolean =>
+        ['PBS', 'BIOLS-AG', 'BIONLS-AG'].some(
+          distribution => course.catalogDistr?.includes(distribution) ?? false
+        ),
+    ],
+    fulfilledBy: 'courses',
+    perSlotMinCount: [1],
+  },
+  {
+    name: 'Additional Requirements',
+    description:
+      'Any course with the Course Distribution PBS, BIOLS-AG, BIONLS-AG, SBA, KCM, MQR, LA, CA, or HA. Language courses may count here.',
+    source:
+      'https://www.human.cornell.edu/sites/default/files/Academics/Registrar/Curriculum%20sheets/2020-2021/HD%20Curriculum%20Sheet%202020-2021.pdf',
+    checker: [
+      (course: Course): boolean =>
+        ['PBS', 'BIOLS-AG', 'BIONLS-AG', 'SBA', 'KCM', 'MQR', 'LA', 'CA', 'HA', 'FL'].some(
```

Comment:
> One requirement (that is the same as DEA) specifically says "Language courses may count here," but the existing DEA checker for this just checked the 'FL' category. However, FL courses don't actually show up right? Should we update these checkers to use the list of language course categories?

Yes, we should use the list of language course categories. See my comment in #454